crates/apub/src/activities/mod.rs

```diff
@@ -231,6 +234,20 @@ pub async fn match_outgoing_activities(
         let creator_id = comment.creator_id;
         CreateOrUpdateNote::send(comment, creator_id, CreateOrUpdateType::Create, context).await
       }
+      SendActivityData::FollowCommunity(follow_community) => {
+        let local_user_view =
+          local_user_view_from_jwt(follow_community.auth.as_ref(), &context).await?;
+        let person = local_user_view.person.clone().into();
+        let community: ApubCommunity =
+          Community::read(&mut context.pool(), follow_community.community_id)
+            .await?
+            .into();
+        if follow_community.follow {
+          Follow::send(&person, &community, &context).await
+        } else {
+          UndoFollow::send(&person, &community, &context).await
+        }
```

Comment:
I would prefer to keep all this code in `crates/apub/src/activities/following/mod.rs` so that this match stays short.